content-src/experiments/no-more-404s.yaml

```diff
@@ -107,7 +107,7 @@ graduation_report: >
   <p><strong>Here's what we learned</strong></p>
   <p>Both teams wanted to get a sense of how many of the URLs that display 404 errors today had pages archived in the Wayback Machine. <a href="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/edit#gid=287298530" target="_blank">We tracked total 404s</a> encountered by Test Pilot participants (1,275,165) as well as results returned by the Archive (185,081). Over the course of the experiment, we averaged a result 14.5 percent of the time, but also observed that over the course of the experiment the daily average rose.</p>
   <figure>
-  <iframe class="dashboard-embed" src="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/pubchart?oid=1805424694&amp;format=interactive"</iframe>
+  <img src="/static/images/experiments/no-more-404s/graduation/gr-001.png" width="100%" height="auto" />
```

Comment:
Not sure that `width` and `height` attributes support percentages and "auto", or if those are more CSS things. I can try re-re-running this through the various HTML validators.